plottr/apps/inspectr.py

```diff
@@ -542,6 +593,40 @@ def plotRun(self, runId: int) -> None:
         }
         win.showTime()
 
+    def setTag(self, item: QtWidgets.QTreeWidgetItem, tag: str):
+        # set tag in the database
+        assert self.filepath is not None
+        runId = int(item.text(0))
+        ds = load_dataset_from(self.filepath, runId)
+        ds.add_metadata('tag', tag)
```

Comment:
i don't think it's fair to reserved ``'tag'`` as the key for tags that inspectr manages. this key is so likely to be used by users outside of inspectr scope and hence i think it's not good idea that when they use inspectr that "tag" can be overwritten by inspectr. also, not that the "interactive experiemnt widget" in qcodes also implements a form of free-form tagging where users can write little notes about the dataset, and for that ds.metadata is also used but the key is not a generic ``"tag"`` but something specific to the "interactive experiemnt widget".
```suggestion
        ds.add_metadata('inspectr_tag', tag)
```

Reply:
Good point, thanks!